packages/field-highlighter/vaadin-field-highlighter.d.ts

```diff
@@ -1 +0,0 @@
-export * from './src/vaadin-field-highlighter.js';
```

Comment:
Let's keep these files just in case, I only meant removing `.d.ts` for styles files.